### PR TITLE
MDCT-1822: Design Feedback Tweeks

### PR DESCRIPTION
**Pull request by ntsummers1**
## Description
Fixes [MDCT-1822](https://qmacbis.atlassian.net/browse/MDCT-1822). MCR: The Padded Edition

Fixes:
* Increase outside margin of the header to 36px (2.25rem)

* Increase outside margin of the footer to 36px (2.25rem)

* Remove Autosave State from subheader

* Decrease size of field hint text to 14px.

### How to test
./dev local
Check out the header in the desktop view and any of the form pages!

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Review by gmrabian — CHANGES_REQUESTED** [review]
Looks good! I have two asks:

1. Convert the px to rem, just to keep with the repo's standard
2. The form runner and the admin banner is now misaligned with the header/footer
main: 
Screen Shot 2022-07-26 at 7 07 06 AM
Screen Shot 2022-07-26 at 7 07 01 AM

This branch:
Screen Shot 2022-07-26 at 7 07 19 AM
Screen Shot 2022-07-26 at 7 07 14 AM
Screen Shot 2022-07-26 at 7 11 09 AM
Screen Shot 2022-07-26 at 7 11 26 AM

**Comment by ntsummers1** [comment]
Asking design how to handle the subbanner now not being aligned. Moving the header also misaligns the side nav here too.

File: services/ui-src/src/components/menus/Sidebar.tsx [hunk 1]
```diff
@@ -149,9 +152,15 @@ const sx = {
     transition: "all 0.3s ease",
     "&.open": {
       marginLeft: "-1rem",
+      "&.desktop": {
+        marginLeft: "-2.3rem",
```
Comment (gmrabian) [inline services/ui-src/src/components/menus/Sidebar.tsx:156]: This one snuck through!

Comment (ntsummers1) [reply services/ui-src/src/components/menus/Sidebar.tsx:156]: Good catch! Fixed

**Comment by ntsummers1** [comment]
> Asking design how to handle the subbanner now not being aligned. Moving the header also misaligns the side nav here too.

Design approved

**Comment by gmrabian** [comment]
Awaiting design feedback on tablet/mobile sidebar misaligned with header/footer

**Comment by ntsummers1** [comment]
Met with Design
- Shrank desktop padding to 2rem
- Aligned all main content and side nav content to same padding as header and footer on mobile and desktop

**Review by gmrabian — APPROVED** [review]
Sweet, sweet alignment

**Pull request merged**